server/src/hardware/protocol/z21/messages.hpp

```diff
@@ -1094,14 +1123,18 @@ struct LanXLocoInfo : LanX
   static constexpr uint8_t flagF0 = 0x10;
   static constexpr uint8_t functionIndexMax = 28;
 
-  uint8_t addressHigh = 0;
-  uint8_t addressLow = 0;
+  static constexpr uint8_t minMessageSize = 7 + 7;
+  static constexpr uint8_t maxMessageSize = 7 + 14;
+
+  uint8_t addressHigh = 0; //db0
+  uint8_t addressLow = 0;  //db1
   uint8_t db2 = 0;
-  uint8_t speedAndDirection = 0;
+  uint8_t speedAndDirection = 0; //db3
   uint8_t db4 = 0;
-  uint8_t f5f12 = 0;
-  uint8_t f13f20 = 0;
-  uint8_t f21f28 = 0;
+  uint8_t f5f12 = 0;    //db5
+  uint8_t f13f20 = 0;   //db6
+  uint8_t f21f28 = 0;   //db7
+  uint8_t f29f31 = 0;   //db8 (firmware >= 1.42)
   uint8_t checksum = 0;
```

Comment:
Shouldn't these be renamed to `db8` and `db9` ? (and add a `checksum() const` method.

When the message is sent from a Z21 with firmware < 1.42 the `f29f31` field contains the checksum and the `checksum` field is then probably undefined or results in a memory access error, this is prone to bugs.

Reply:
I don't think we need a checksum method because we can already use `LanX::isChecksumValid(const LanX &lanX)` which has a special case for this message.
The `checksum` now renamed `db9` is not accessed anywhere